communications.md

```diff
@@ -0,0 +1,68 @@
+
+As soon as you are _not_ obligated, and/or interested in any of them, GTFO.
+
+As for emails, since its innate opt-out (or can't even) nature, you may do the following:
+
+- Pin/star items that need future attention. Archive them as soon as they're done, or no longer needed.
+- Optionally snooze items that need future attention. Archive them as soon as they're done, or no longer needed.
+- Archive the rest.
+
+## Asynchronous by Default
+
+Whenever you feel the need to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and the fact that they may be in the "zone" and should not be interrupted if you are at all considerate about others and their productivity.
+
+Try to communicate in such order:
+
+1. Send a well-written message in a relevant Slack channel (or Github issue, or Asana project).
+2. If this needs a certain person's or group's attention, elevate its priority by pinging them right below the message, or start a thread and ping them there.
+3. A rare but not impossible reactor-meltdown level of urgency, yell out in any synchronous manner and get people's attention.
+
+## Text by Default
+
+Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, asynchronously send this message.
+
+## Transparent by Default
+
+Don't be embarrassed about asking "stupid" questions. Try to always write your messages in a publically accessible (or at a bare minimum, team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
+
+A few exceptions:
+
+- When you are communicating work-related credentials or secrets, use direct messages, or very well-controlled private channels.
+- When you are talking about subjects that are irrelevant to work, use direct messages (but to be honest, wouldn't one of the social media/chat apps fit better?).
+
+## Budget Communication Time
+
+For asynchronous communications, try to allocate a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for our job, which has an innate quality of being a creative endeavor.
+
+For synchronous communications:
+
+1. Aggressively timebox them. If a meeting was set to be an hour-long, ask if it can be shortened to 30 minutes; if it was set to be 30 minutes long, ask if it can be shortened to 15 minutes...
+2. Write down the agenda, and prepare every participant.
+3. Instead of abstractly talk about a problem in a hollow boardroom, demonstrate the problem, perhaps on-site or screen-shared (which can be your local branch of a project that you're working on). Do not stretch beyond the agreed timebox, make a decision and move on.
```

Comment:
We could also add:

4. Ensure there are clear and concise take-aways. Every meeting must have a clear outcome
5. One of the individuals attending should concisely summarize the meeting and key points of the meeting. These notes should be posted on an appropriate channel for both participants and anyone else who might be interested in the contents but couldn't attend/join the meeting.
6. Each meeting is the collective productivity value of all its participants. Consider each participant to be on average contributing $200/hr; it is always good to judge the need/value of each meeting against the economic value of the meeting (what other productive task could these participants be doing?)